linux/tests/threadpool_linux_int/threadpool_linux_int.c

```diff
@@ -103,6 +104,7 @@ static void threadpool_long_task(void* context)
     WRAP_DATA* data = context;
     ASSERT_ARE_EQUAL(int, 0, strcmp(data->mem, "READY"));
     strcpy(data->mem, "DONE");
+    ThreadAPI_Sleep(1);
```

Comment:
>ThreadAPI_Sleep(1); [](http://example.com/codeflow?start=4&length=19)

Why is this Sleep needed?

Reply:
It was creating a raise condition and the counter would stop 4998. Similar to threadpool_long_task_v2, it is solving the problem for this test.

Reply:
Resolved by changing Sleep to Shared locks.